kinds/dashboard/dashboard_kind.cue

```diff
@@ -97,39 +105,72 @@ lineage: seqs: [
 				///////////////////////////////////////
 				// Definitions (referenced above) are declared below
 
-				// TODO docs
+				// TODO: this should be a regular DataQuery that depends on the selected dashboard
+				// these match the properties of the "grafana" datasouce that is default in most dashboards
 				#AnnotationTarget: {
-					limit:    int64
+					// Only required/valid for the grafana datasource... 
```

Comment:
we could potentially model this special case directly, here in schema - but it would be preferable to refactor the grafana datasource to avoid such hacks